.rat-excludes

```diff
@@ -4,6 +4,7 @@
 **/*.md
 **/*.iml
 **/target/**
+**/dist/**
```

Comment:
Directory `ams/dist` contains some files that need to be checked, we may not ignore them all.

Reply:
make sense, some files will be added after building project. I will only exclude the font files